src/kriscv/kdist/riscv-semantics/lemmas/sparse-bytes-simplifications.md

```diff
@@ -18,16 +18,16 @@ For symbolic execution, we need to tackle the patterns of `#bytes(B +Bytes _) _`
   
   rule pickFront(#bytes(substrBytes(B, I0, N) +Bytes _) _ , I1) => substrBytes(substrBytes(B, I0, N), 0, I1)
```

Comment:
Would it make sense to use `substrBytes(X, 0, N)` instead of `pickFront(X, N)`? Then these lemmas would be covered by a simple lemma about `substrBytes` composition.

Reply:
I've just realized these functions are defined over different sorts, so never mind.